mypy/stubtest.py

```diff
@@ -205,7 +205,7 @@ def test_module(module_name: str) -> Iterator[Error]:
 
     try:
         runtime = silent_import_module(module_name)
-    except Exception as e:
+    except BaseException as e:
```

Comment:
Is it worth special-casing `KeyboardInterrupt`?

```suggestion
    except KeyboardInterrupt:
        raise 
    except BaseException as e:
```

Reply:
I thought about it, but wasn't sure. I guess we've made it this far without libraries raising KeyboardInterrupt, so might as well

Reply:
Libraries that manually raise `KeyboardInterrupt` in the global scope don't deserve to have stubs written for them

Reply:
I kind of thought that way about SystemExit tbh